### PR TITLE
Navbar layout

### PR DESCRIPTION
**Pull request by gsaslis**
Please see commit comments for particular changes included in this PR

### CODE REVIEW

File: Rocket.Chat.iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -564,7 +564,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				INFOPLIST_FILE = Rocket.Chat.iOS/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (rafaelks) [inline Rocket.Chat.iOS.xcodeproj/project.pbxproj]: Nice! :-)

Comment (gsaslis) [inline Rocket.Chat.iOS.xcodeproj/project.pbxproj]: Not sure about this after all... Having second thoughts... What do you guys think? 
should we start from iOS8?
@RocketChat/owners

Comment (rafaelks) [inline Rocket.Chat.iOS.xcodeproj/project.pbxproj]: @gsaslis I don't think so... the same devices that support iOS 8 will support iOS 9. Also, iOS 9 have a lot of cool features that we can use by targeting it.

Comment (rodrigok) [inline Rocket.Chat.iOS.xcodeproj/project.pbxproj]: I think we can use iOS9 as base target.

Comment (gsaslis) [inline Rocket.Chat.iOS.xcodeproj/project.pbxproj]: great! :) :balloon:

**Comment by gsaslis** [comment]
can someone please review and merge this in?

**Pull request merged**